common/predictive-text/build.sh

```diff
@@ -45,13 +45,32 @@ build ( ) {
   build-worker && wrap-worker && build-main
 }
 
+# Build the worker and the main script for headless use.
+build_headless ( ) {
```

Comment:
Style nitpick:

```suggestion
build-headless () {
```

(This is consistent with the rest of the build script.)